modules/loader/src/main/java/com/opengamma/strata/loader/csv/FullSwapTradeCsvLoader.java

```diff
@@ -0,0 +1,674 @@
+  private static final String COMPOUNDING_METHOD_FIELD = "Compounding Method";
+
+  private static final String NOTIONAL_CURRENCY_FIELD = "Notional Currency";
+  private static final String NOTIONAL_INITIAL_EXCHANGE_FIELD = "Notional Initial Exchange";
+  private static final String NOTIONAL_INTERMEDIATE_EXCHANGE_FIELD = "Notional Intermediate Exchange";
+  private static final String NOTIONAL_FINAL_EXCHANGE_FIELD = "Notional Final Exchange";
+  private static final String FX_RESET_INDEX_FIELD = "FX Reset Index";
+  private static final String FX_RESET_RELATIVE_TO_FIELD = "FX Reset Relative To";
+  private static final String FX_RESET_OFFSET_DAYS_FIELD = "FX Reset Offset Days";
+  private static final String FX_RESET_OFFSET_CAL_FIELD = "FX Reset Offset Calendar";
+  private static final String FX_RESET_OFFSET_ADJ_CNV_FIELD = "FX Reset Offset Adjustment Convention";
+  private static final String FX_RESET_OFFSET_ADJ_CAL_FIELD = "FX Reset Offset Adjustment Calendar";
+
+  private static final String INITIAL_STUB_RATE_FIELD = "Initial Stub Rate";
+  private static final String INITIAL_STUB_AMOUNT_FIELD = "Initial Stub Amount";
+  private static final String INITIAL_STUB_INDEX_FIELD = "Initial Stub Index";
+  private static final String INITIAL_STUB_INTERPOLATED_INDEX_FIELD = "Initial Stub Interpolated Index";
+  private static final String FINAL_STUB_RATE_FIELD = "Final Stub Rate";
+  private static final String FINAL_STUB_AMOUNT_FIELD = "Final Stub Amount";
+  private static final String FINAL_STUB_INDEX_FIELD = "Final Stub Index";
+  private static final String FINAL_STUB_INTERPOLATED_INDEX_FIELD = "Final Stub Interpolated Index";
+  private static final String RESET_FREQUENCY_FIELD = "Reset Frequency";
+  private static final String RESET_DATE_CNV_FIELD = "Reset Date Convention";
+  private static final String RESET_DATE_CAL_FIELD = "Reset Date Calendar";
+  private static final String RESET_METHOD_FIELD = "Reset Method";
+  private static final String FIXING_RELATIVE_TO_FIELD = "Fixing Relative To";
+  private static final String FIXING_OFFSET_DAYS_FIELD = "Fixing Offset Days";
+  private static final String FIXING_OFFSET_CAL_FIELD = "Fixing Offset Calendar";
+  private static final String FIXING_OFFSET_ADJ_CNV_FIELD = "Fixing Offset Adjustment Convention";
+  private static final String FIXING_OFFSET_ADJ_CAL_FIELD = "Fixing Offset Adjustment Calendar";
+  private static final String NEGATIVE_RATE_METHOD_FIELD = "Negative Rate Method";
+  private static final String FIRST_RATE_FIELD = "First Rate";
+  private static final String ACCRUAL_METHOD_FIELD = "Accrual Method";
+  private static final String RATE_CUT_OFF_DAYS_FIELD = "Rate Cut Off Days";
+  private static final String INFLATION_LAG_FIELD = "Inflation Lag";
+  private static final String INFLATION_METHOD_FIELD = "Inflation Method";
+  private static final String INFLATION_FIRST_VALUE_FIELD = "Inflation First Value";
```

Comment:
My feeling is to stick with the names used in Strata. "Initial Index Level" is the name in FpML FYI.